### ISSUE TITLE
Browse under Verify Video Capture isn't correct

### ISSUE THREAD

**Issue by SoOlditHurts** [MerlinCooper/iRacingReplayDirector#87]
The Browse feature under Race Capture > Verify Video Capture doesn't produce a usable browsing experience. The Browse under Video Encoding > Verify Video works correctly.

### Non-Working

![Screenshot 2023-01-28 113217](https://user-images.githubusercontent.com/83684120/215282008-a2f99834-79a0-4906-8a4b-208367ca0ff5.jpg)

### Working

![Screenshot 2023-01-28 113734](https://user-images.githubusercontent.com/83684120/215282268-c3eb473f-b1e4-494f-8c8d-1fb162627c8f.jpg)

**Comment by MerlinCooper** [comment]
Interesting - both "Browse" buttons do work on my side the same way. 
![grafik](https://user-images.githubusercontent.com/3088879/215313078-8efe3d03-889a-460d-bff2-e1acc4a66bc4.png)

Did you type in the directory name manually - looks like you are using '\' and in my case it's '/'

**Comment by SoOlditHurts** [comment]
I found the problem. I'm on W11 22H2 and there is a bug between it and OneDrive that is the cause.  If I don't back up my Desktop using OneDrive, then the Browse for Folder functions correctly.

I copied the folder path from the Video Encoding tab and pasted that into the Video Capture Folder. Windows path uses "\" (forward slash) in normal use. However "/" can be used interchangeably with it. It's a very old UNIX leftover from when they created it.
